<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="firstNum">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>first number</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondNum">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>second number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtfirstNum">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtsecondNum">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="plusBTN">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minusBTN">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mulBTN">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="devideBTN">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelResBTN">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>The Result is </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtResults">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
